src/WebJobs.Script.WebHost/App_Start/WebApiConfig.cs

```diff
@@ -72,6 +83,33 @@ public static void Register(HttpConfiguration config, WebHostSettings settings =
             config.InitializeReceiveSalesforceWebHooks();
         }
 
+        private static void PrependFoldersToEnvironmentPath()
+        {
+            string home = Environment.GetEnvironmentVariable("HOME");
```

Comment:
We could, but note that the logic actually uses HOME beyond detection. So we'd really need to add HomeFolder to `WebHostSettings`. And note sure is that would push changes in a bunch of other places.